tests/Unit/RequestMoneyValidatorTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\src\Deviation\Deviation;
+use App\src\RequestMoneyValidator\Dto\RequestDto;
+use App\src\RequestMoneyValidator\Dto\TransactionDto;
+use App\src\RequestMoneyValidator\Services\RequestMoneyValidator;
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use Mockery as m;
+
+class RequestMoneyValidatorTest extends TestCase
```

Comment:
Good job with testing, although I expected to see more test cases besides the ones I have put as a sample for a task understanding.